modules/metering-exposing-the-reporting-api.adoc

```diff
@@ -25,6 +25,7 @@ METERING_ROUTE_HOSTNAME=$(oc -n openshift-metering get routes metering -o json |
 
 Next, set up authentication using either a service account token or basic authentication with a username/password.
 
+[id="metering-authenticate-using-service-account_{context}"]
```

Comment:
What is this module supposed to be? It's not a well-formed procedure module - that would have a single heading and a clear procedure section. Only procedure modules are allowed to have gerunds in the module title.

Reply:
Good point. Upstream you had some options around how to expose the reporting API, but the need to cover those goes away on OS b/c the route is automatically exposed. I'll rework this, I think to be "Accessing the reporting API", and then I'll update the ensuing content to be more procedural.